src/main/java/org/folio/inventory/dataimport/handlers/actions/MarcBibMatchedPostProcessingEventHandler.java

```diff
@@ -39,6 +43,7 @@ public class MarcBibMatchedPostProcessingEventHandler implements EventHandler {
   private static final String ERROR_HOLDING_MSG = "Error loading inventory holdings for MARC BIB";
   private static final String MATCHED_MARC_BIB_KEY = "MATCHED_MARC_BIBLIOGRAPHIC";
   private final Storage storage;
+  public static final String CENTRAL_TENANT_ID = "CENTRAL_TENANT_ID";
```

Comment:
Looks like the constant might be private

Reply:
Thanks, done